### PR TITLE
compile error fixes - updated to RL 1.9.7

### PR DESCRIPTION
**Pull request by zebdo**

### CODE REVIEW

**Comment by zebdo** [comment]
please tell me how shit my additions are; i want to grow

File: src/main/java/net/runelite/rsb/wrappers/client_wrapper/RSClient.java [hunk 1]
```diff
@@ -140,6 +147,30 @@ public Widget getScriptDotWidget() {
...
+    public void setCameraPitchTarget(int cameraPitchTarget) {}
+
+    @Override
+    @Deprecated
+    public RenderOverview getRenderOverview() {
+        return null;
+    }
+
+    @Override
+    public WorldMap getWorldMap() {
```
Comment (electric-lemonade) [inline src/main/java/net/runelite/rsb/wrappers/client_wrapper/RSClient.java:170]: Why leave these overrides in when the wrapper has proper implementations?

File: src/main/java/net/runelite/rsb/wrappers/client_wrapper/BaseClientWrapper.java [hunk 2]
```diff
@@ -885,6 +893,9 @@ public void setStringStackSize(int stackSize) {
         wrappedClient.setStringStackSize(stackSize);
     }
 
+    @Override
+    public void setCameraPitchTarget(int cameraPitchTarget) { wrappedClient.setCameraPitchTarget(cameraPitchTarget); }
```
Comment (electric-lemonade) [inline src/main/java/net/runelite/rsb/wrappers/client_wrapper/BaseClientWrapper.java:897]: The spacing on this is inconsistent with the rest

File: src/main/java/net/runelite/rsb/wrappers/client_wrapper/BaseClientWrapper.java [hunk 3]
```diff
@@ -182,11 +184,16 @@ public int getCameraPitch() {
...
 
+    @Override
+    public int getCameraPitchTarget() {return wrappedClient.getCameraPitchTarget(); }
     @Override
     public int getCameraYaw() {
         return wrappedClient.getCameraYaw();
     }
 
+    @Override
+    public int getCameraYawTarget() {return wrappedClient.getCameraYawTarget(); }
```
Comment (electric-lemonade) [inline src/main/java/net/runelite/rsb/wrappers/client_wrapper/BaseClientWrapper.java:195]: These two also have inconsistent spacing

File: src/main/java/net/runelite/rsb/wrappers/client_wrapper/BaseClientWrapper.java (hunk 2 above)
Comment (electric-lemonade) [inline src/main/java/net/runelite/rsb/wrappers/client_wrapper/BaseClientWrapper.java:897]: Seems like `setCameraYawTarget` is missing

Comment (zebdo) [reply src/main/java/net/runelite/rsb/wrappers/client_wrapper/BaseClientWrapper.java:897]: my bad, just realized intellij auto collapses braces, i just copied what I saw; will be fixed in next commit

File: src/main/java/net/runelite/rsb/wrappers/client_wrapper/RSClient.java (hunk 1 above)
Comment (zebdo) [reply src/main/java/net/runelite/rsb/wrappers/client_wrapper/RSClient.java:170]: oversight, thanks for alerting me

File: src/main/java/net/runelite/rsb/wrappers/client_wrapper/BaseClientWrapper.java (hunk 2 above)
Comment (zebdo) [reply src/main/java/net/runelite/rsb/wrappers/client_wrapper/BaseClientWrapper.java:897]: it already exists

Comment (electric-lemonade) [reply src/main/java/net/runelite/rsb/wrappers/client_wrapper/BaseClientWrapper.java]: Ah I see, I was comparing with my local changes and I had moved it to be next to `setCameraPitchTarget`

File: src/main/java/net/runelite/rsb/wrappers/client_wrapper/BaseClientWrapper.java [hunk 4]
```diff
@@ -182,11 +184,18 @@ public int getCameraPitch() {
         return wrappedClient.getCameraPitch();
     }
 
+    @Override
+    public int getCameraPitchTarget() {return wrappedClient.getCameraPitchTarget(); }
```
Comment (electric-lemonade) [inline src/main/java/net/runelite/rsb/wrappers/client_wrapper/BaseClientWrapper.java:188]: This one is still missing spacing

**Review by GigiaJ — APPROVED** [review]
I have finished reviewing. Good work man.

**Pull request merged**